project.clj

```diff
@@ -85,8 +94,11 @@
                                   [prismatic/schema-generators "0.1.0"
                                    :exclusions [prismatic/schema]]]
                    :resource-paths ["model"
-                                    "test/resources"]}
-
+                                    "test/resources"]
+                   :jvm-opts ["-Dcom.sun.management.jmxremote"
+                              "-Dcom.sun.management.jmxremote.ssl=false"
+                              "-Dcom.sun.management.jmxremote.authenticate=false"
+                              "-Dcom.sun.management.jmxremote.port=5555"]}
```

Comment:
In fact I needed them to try to fix some bug, I added them in the commit.

It is not required for the metrics to work. I'll remove them.